features/step_definitions/catalogue_steps.rb

```diff
@@ -117,14 +117,27 @@
 Then(/^I am taken to page (\d+) of results$/) do |page_number|
   page_number = page_number.to_i
   expect(current_url).to include("page=#{page_number}")
-  expect(page).to have_selector(:xpath, "//a[contains(text(), 'Next')]//following-sibling::span[contains(text(),'page')]")
-  expect(page).to have_selector(:xpath, "//a[contains(text(), 'Next')]//following-sibling::span[contains(text(),'page')]/..//following-sibling::span[@class='page-numbers'][contains(text(), '#{page_number + 1} of')]")
-  if page_number == 1
-    expect(page).to have_selector(:xpath, "//a[contains(text(), 'Next')]//following-sibling::span[contains(text(),'page')]")
-    expect(page).not_to have_selector(:xpath, "//a[contains(text(), 'Previous')]//following-sibling::span[contains(text(),'page')]")
+
+  if page.has_css?('.dm-pagination') # @TODO: New pagination - remove else clause when search page released
+    expect(page).to have_selector(:xpath, "//nav//ul//li//a//span[contains(text(), 'Next page')]")
```

Comment:
What happens if the number of results are small? Will we always have a next page?

Reply:
Hmm, I'm not sure we can necessarily guarantee it, but this test is used on the list of services, which numbers in the thousands, and the list of suppliers, which is similarly massive.

The test I can see possibly failing in the future is 'There is no pagination on the list of suppliers page if there is less than or equal to 100 results', which just loads the suppliers beginning with Q, which currently number around 58. We could look at mocking that somehow if it becomes an issue.